store/tikv/coprocessor.go

```diff
@@ -37,7 +37,12 @@ type CopClient struct {
 func (c *CopClient) SupportRequestType(reqType, subType int64) bool {
 	switch reqType {
 	case kv.ReqTypeSelect:
-		return supportExpr(tipb.ExprType(subType))
+		switch subType {
+		case kv.ReqSubTypeAgg:
```

Comment:
Suggest to remove `ReqSubTypeAgg` and use corresponding `exprType` directly to check.